src/Presentation/Console/Commands/UpdateCommand.php

```diff
@@ -129,4 +122,14 @@ protected function checkForUpdate(OutputInterface $output): void
             $output->writeln($message->getMessage(), OutputInterface::VERBOSITY_VERBOSE);
         }
     }
+
+    /**
+     * @return void
+     */
+    protected function createDatabase(): void
```

Comment:
The same code is in the InitCommand. Please, extract it to avoid code duplicaction?

Reply:
it's not duplicate logic. it's a call the command